src/openvic-simulation/dataloader/Dataloader.cpp

```diff
@@ -1135,18 +1138,38 @@ bool Dataloader::load_defines(
 		spdlog::critical_s("Failed to load bookmarks!");
 		ret = false;
 	}
-	if (!definition_manager.get_country_definition_manager().load_countries(
-		definition_manager, *this, parse_defines(lookup_file(countries_file)).get_file_node()
-	)) {
-		spdlog::critical_s("Failed to load countries!");
-		ret = false;
-	}
-	if (!definition_manager.get_country_definition_manager().load_country_colours(
-		parse_defines(lookup_file(country_colours_file)).get_file_node()
-	)) {
-		spdlog::critical_s("Failed to load country colours!");
-		ret = false;
+
+	{
+		CountryParser country_parser {
+			definition_manager.get_politics_manager().get_government_type_manager().get_government_type_registry(),
+			definition_manager.get_pop_manager().get_culture_manager().get_graphical_culture_type_registry(),
+			definition_manager.get_military_manager().get_unit_type_manager().get_unit_type_registry(),
+			definition_manager.get_politics_manager().get_issue_manager().get_party_policy_group_registry(),
+			definition_manager.get_politics_manager().get_issue_manager().get_party_policy_registry(),
+			definition_manager.get_politics_manager().get_ideology_manager().get_ideology_registry(),
+			parse_defines(lookup_file(countries_file))
+		};
```

Comment:
```suggestion
		};

```
New line to separate it from the if else chain.